### PR TITLE
Patch Dia

### PR DESCRIPTION
**Pull request by szhu**
http://sourceforge.net/p/dia-installer/support-requests/98/

### CODE REVIEW

**Comment by vitorgalvao** [comment]
That page is a 404. What does this fix?

File: Casks/dia.rb [hunk 1]
```diff
@@ -9,4 +9,8 @@
   license :gpl
 
   app 'Dia.app'
+  postflight do
```
Comment (vitorgalvao) [inline Casks/dia.rb]: Needs a blank line before.

**Comment by szhu** [comment]
The app doesn't start on 10.10+ unless this patch is made.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 744ff9286621f1189518d72eef2dd31d0d5d8e1d. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by szhu** [comment]
I am on mobile so I couldn't amend. Thanks for doing that for me!

**Comment by vitorgalvao** [comment]
No problem, thank you for the continued contributions.

**Comment by szhu** [comment]
One more question: This patch is pretty risky and I'd rather include a diff-style patch and have postflight apply it. I wasn't sure how to do that without committing a new file. Are there any precedents for this?

**Comment by vitorgalvao** [comment]
> Are there any precedents for this?

I think not.

**Pull request closed without merging**